client/lib/route/page-notifier.js

```diff
@@ -1,23 +1,34 @@
 /*eslint no-multi-spaces: 0*/
 // node dependency
 var EventEmitter   = require( 'events' ).EventEmitter;
+var ipc = require( 'electron' ).ipcRenderer;
+var includes = require( 'lodash' ).includes;
 
 // internal dependency
-var page           = require( 'page' );
-var Route          = page.Route;
+var page		   = require( 'page' );
+var Route		  = page.Route;
 
 // routing state and middleware
-var location       = new EventEmitter();
+var location	   = new EventEmitter();
 var currentContext = null;
-var route          = new Route( '*' );
+var commandLine	= {};
+var route		  = new Route( '*' );
 
 // Inject our route at the front so we are always called
 page.callbacks.unshift( route.middleware( function( context, next ) {
+	context.commandLine = commandLine;
 	next();
 	location.emit( 'change', context, currentContext );
 	currentContext = context;
 } ) );
 
+ipc.on( 'args', function( sender, args ) {
```

Comment:
This seems like something that could be better in the `lib/desktop/index.js` file along with the rest of the IPC listeners. I can see why you've got it here so you can pass along the command line. Maybe the page notifier can be passed the value?